src/data/models/Settings.ts

```
@@ -0,0 +1,37 @@
+    triggerOnlyOnNewLine: boolean;
+    queryPattern: RegExp;
+    bindings: SlashCommand[];
+}
+
+export interface MenuSuggestion {
+    pair: SlashCommand;
+    element: HTMLDivElement;
+}
+
+export type EnhancedEditor = Editor & {
+    cursorCoords: Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)